middleware-catalog-module/catalog-dao/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
```

Comment:
All classes on directory /target must be in .ignore. Now we need to delete them to remove from repository. After deleting do git add, git commit, git push.